### ISSUE TITLE
Add a new Extension method on Result instead of Controller to map to an ActionResult

### ISSUE THREAD

**Issue by ardalis** [ardalis/Result#59]
Basically instead of:

```
public override ActionResult<IEnumerable<WeatherForecast>> Handle(ForecastRequestDto request)
{
    return this.ToActionResult(_weatherService.GetForecast(request));
}
```

do this:
```
public override ActionResult<IEnumerable<WeatherForecast>> Handle(ForecastRequestDto request)
{
     return _weatherService.GetForecast(request).ToActionResult(this);
}
```

**Comment by ardalis** [comment]
@topswagcode it's all yours

**Comment by TopSwagCode** [comment]
I will take a look at it

**Comment by JelmarVA** [comment]
This Issue was implemented in #66
